test/helpers/fixtures.js

```diff
@@ -32,7 +33,8 @@ async function initRouterContracts() {
     await tokens.DAI.approve(uniswapv2, ether('1'));
     await tokens.DAI.approve(uniswapv3, ether('1'));
     await tokens.DAI.approve(paraswap, ether('1'));
-    await tokens.DAI.approve(uniswapUniversalRouter, ether('1'));
+    await tokens.DAI.approve(permit2, ether('1'));
+    await permit2.approve(tokens.DAI.target, uniswapUniversalRouter.target, ether('1'), Date.now());
```

Comment:
This approval seems redundant because we don't have tests for cases with `permit2` yet, and it could be misleading to imply that we use it.